app/assets/stylesheets/content/_forms.sass

```diff
@@ -381,7 +381,8 @@ fieldset.form--fieldset
 
 .form--select
   @extend %input-style
-  line-height: 1.5
+  line-height: normal
+  padding: 3px 24px 3px 3px
```

Comment:
@vcostin  we're generally using `rem` rather than pixel values. `rem-calc` may help, if need be.

Please take a look at `frontend/bower_components/foundation-apps/scss/components/_forms.scss`

What do you think about adjusting `$form-padding` slightly?